lab_1/main.py

```diff
@@ -14,12 +14,14 @@ def tokenize(text: str) -> list or None:
     """
     if not isinstance(text, str):
         return None
+    if "\n" in text:
```

Comment:
if `\t` is in a text? :)
maybe consider using `isalpha` and `isspace`

Reply:
Oh, I didn't know about isspace. Thanks, it makes things much easier)0